src/components/MyNdla/Folder.tsx

```diff
@@ -171,12 +171,13 @@ interface Props {
   type?: LayoutType;
   menu?: ReactNode;
   folder: GQLFolder;
-  foldersCount: Record<string, FolderTotalCount>;
-  isFolder: boolean;
+  foldersCount?: FolderTotalCount;
```

Comment:
Hva er grunnen til denne endringen?

Reply:
Så ikke logikken med å sende ned hele foldersCount objektet generert høyere opp når du kun bruker resultatet for den mappen du er på, såååå ryddigere kode 🥉